gamma_transients/lib/processing.py

```diff
@@ -0,0 +1,1253 @@
+    tab[colname][0].pop()
+
+
+def radec_to_SimbadCircle(ra: float, dec: float, radius_deg=0.2) -> str:
+    """
+    Return a simbad criteria string for a circle around a SkyCoord
+
+    Parameters
+    ----------
+    ra_deg: float
+        Right ascension in degrees
+
+    dec_deg: float
+        Declination in degrees
+
+    radius_deg : float
+        Radius of the circle in degrees
+
+    Returns
+    -------
+    str
+        Simbad criteria string
+    """
+    prefix = ""
+    if dec >= 0:
+        prefix += "+"
+    return f"region(circle,{ra} {prefix}{dec},{radius_deg}d)"
+
+
+def cat2hpx(
+    lon: np.ndarray, lat: np.ndarray, nside: int, radec: bool = True
+) -> np.ndarray:
+    """
+    Convert a catalogue to a HEALPix map of number counts per resolution
+    element. (https://stackoverflow.com/a/50495134)
+
+    Parameters
+    ----------
+    lon, lat : (ndarray, ndarray)
+        Coordinates of the sources in degree. If radec=True, assume input is in the fk5
+        coordinate system. Otherwise assume input is glon, glat
+
+    nside : int
+        HEALPix nside of the target map
+
+    radec : bool
+        Switch between R.A./Dec and glon/glat as input coordinate system.
+
+    Return
+    ------
+    hpx_map : ndarray
+        HEALPix map of the catalogue number counts in Galactic coordinates
+
+    """
+
+    npix = hp.nside2npix(nside)
+
+    if radec:
+        eq = SkyCoord(lon, lat, frame="fk5", unit="deg")
+        l, b = eq.galactic.l.value, eq.galactic.b.value
+    else:
+        l, b = lon, lat
+
+    # conver to theta, phi
+    theta = np.radians(90.0 - b)
+    phi = np.radians(l)
+
+    # convert to HEALPix indices
+    indices = hp.ang2pix(nside, theta, phi)
+
+    idx, counts = np.unique(indices, return_counts=True)
+
+    # fill the fullsky map
+    hpx_map = np.zeros(npix, dtype=int)
+    hpx_map[idx] = counts
+
+    return hpx_map
+
+
+class Multiplets:
+    """
+    Class for handling multiplets
+
+    Attributes
+    ----------
+    paths : list[str]
+        List of paths to the multiplet folders
+
+    table : astropy.table.Table
+        Table of the multiplets
+
+        colnames:
+        'Nmax'
+        'OBS_ID'
+        'ID'
+        'RA'
+        'DEC'
+        'TIME'
+        'ENERGY'
+        'dt'
+        'da'
+
+        (added by Multiplets.addCoordinateInfo()
+        'MEDIAN_RA'
+        'MEDIAN_DEC'
+        'SkyCoord'
+        'MEDIAN_GLAT'
+        'MEDIAN_GLON'
+
+        (added by Multiplets.searchTeVCat()
+        'TEVCAT_SOURCE_NAME'
+        'TEVCAT_SOURCE_TYPE'
+        'TEVCAT_DISTANCES_DEG'
+
+        (added MANUALLY, use in_which_container())
+        'DS_INDEX'
+
+        (added by Multiplets.addRMS()
+        'ANGULAR_MEASURE_DEG'
+
+
+    reduced : astropy.table.Table
+        Table of the reduced multiplets
+
+
+    """
+
+    def __init__(self, path: str) -> None:
+        self.paths = [path]
+        self.loadMpletsBare(path)
+
+    def loadMpletsBare(self, path) -> None:
```

Comment:
here and below: follow PEP8 conventions on functions naming

Reply:
done